t/app/model/responsepriority.t

```diff
@@ -0,0 +1,97 @@
+use strict;
+use warnings;
+use Test::More;
+
+use FixMyStreet::App;
+use FixMyStreet::TestMech;
+use JSON::MaybeXS;
+
+my $mech = FixMyStreet::TestMech->new;
+my $area_id = 123;
+
+my $oxfordshire = $mech->create_body_ok(2237, 'Oxfordshire County Council', id => 2237);
+my $potholes_contact = $mech->create_contact_ok( body_id => $oxfordshire->id, category => 'Potholes', email => 'potholes@example.com' );
+my $traffic_lights_contact =$mech->create_contact_ok( body_id => $oxfordshire->id, category => 'Traffic lights', email => 'lights@example.com' );
+
+my $potholes_response_priority = FixMyStreet::App->model('DB::ResponsePriority')->find_or_create(
+    {
+        body_id => 2237,
+        name => 'Potholes',
+        description => 'This priority is to do with potholes'
+    }
+);
+$potholes_response_priority->contact_response_priorities->find_or_create({
+    contact_id => $potholes_contact->id,
+});
+
+my $general_response_priority = FixMyStreet::App->model('DB::ResponsePriority')->find_or_create(
+    {
+        body_id => 2237,
+        name => 'All categories',
+        description => 'This priority is for all categories'
+    }
+);
+$general_response_priority->contact_response_priorities->find_or_create({
+    contact_id => $potholes_contact->id,
+});
+$general_response_priority->contact_response_priorities->find_or_create({
+    contact_id => $traffic_lights_contact->id,
+});
+
+FixMyStreet::App->model('DB::BodyArea')->create( { body => $oxfordshire, area_id => $area_id } );
```

Comment:
Same as with other tests, this line shouldn't be needed, I think, remove the ID from Oxfordshire instead to test different body/area ID.